testsuite/MDAnalysisTests/utils/test_modelling.py

```diff
@@ -253,8 +251,8 @@ def test_merge_with_topology(self):
         # One of these bonds isn't in the merged Universe
         assert_(len(ag2[0].bonds) - 1 == len(u2.atoms[20].bonds))
 
-    def test_merge_with_topology_from_different_universes(self):
-        u3 = MDAnalysis.Merge(self.u.atoms[:110], self.u2.atoms)
+    def test_merge_with_topology_from_different_universes(self, u, u2):
+        u3 = MDAnalysis.Merge(u.atoms[:110], u2.atoms)
```

Comment:
If `u1` and `u2` are moved and renamed, this cannot remain `u3`. Maybe `u_merge`?